tests/training/test_imputation.py

```diff
@@ -6,25 +6,29 @@
 def test_imputer():
     input_df = pd.DataFrame({
         'col1': [10, 13, 10],
-        'col2': [50, 100, None]
+        'col2': [50, 100, None],
```

Comment:
Instead of changing the test `test_imputer` I would suggest creating a new one, something like `test_imputer_with_fill_value`. 

This way we would have tests testing both scenarios.